docs/timeseries/io.rst

```diff
@@ -197,6 +197,19 @@ To write data held in any of the :mod:`gwpy.timeseries` classes to a GWF file, s
 
 **If the output file already exists it will be overwritten.**
 
+.. note::
+
+   When writing a timeseries to a `gwf` file, the `TimeSeries.name` property is used to determine the `channel`. Therefore, if you want to write a file and then read it back in, you should first set the name to the chosen channel, e.g.
+
+   .. code-block:: python
+
```

Comment:
Can this code block _define_ a `TimeSeries`, so that the reader understands a circumstance where the `name` property is unassigned?